homeassistant/components/uptimerobot/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: no actions
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: no actions
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: no events
+  entity-unique-id: done
+  has-entity-name: done
```

Comment:
why is `_attr_name = None` in the base entity?

Reply:
why not if it's a common pattern ?

Reply:
IMHO, there is no other clean way to name entities here:

![image](https://github.com/user-attachments/assets/a14195a5-f86e-484e-bb6b-981e3f012d45)

Reply:
UptimeMonitor is a monitoring service, so "on" means monitored, "off" not monitored.